### PR TITLE
Complete DOM Manipulation Challenge Exercises

### PR DESCRIPTION
**Pull request by mosk133**
This pull request includes the completion of all DOM manipulation exercises.

### CODE REVIEW

**Comment by mosk133** [comment]
<span aria-hidden="true">😎</span> Deploy Preview: [https://01-dom-manipulation-guillem.netlify.app](https://01-dom-manipulation-guillem.netlify.app)

File: index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+// Select the element with the ID 'mainTitle' and log it to the console.
+
+let mainTitle = document.getElementById("mainTitle");
+console.log(mainTitle);
+
+// EXERCISE 2: Changing HTML Content
+// Select all paragraph elements and change the content of the second paragraph to "This paragraph has been changed."
+
+let secondParagraph = document.getElementsByTagName("p")[1];
+secondParagraph.innerHTML = "<p>This paragraph has been changed.</p>";
```
Comment (tictools) [inline index.js:11]:
```suggestion
secondParagraph.textContent = "<p>This paragraph has been changed.</p>";
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+setTimeout(function () {
+    firstParagraph.classList.remove("highlight");
+}, 3000);
+
+// EXERCISE 5: Creating and Appending Elements
+// Create a new list item with the text "Item 4" and append it to the existing list.
+
+let newItem4 = document.createElement("li");
+newItem4.textContent = "Item 4";
+let ul = document.getElementById("list");
```
Comment (tictools) [inline index.js:35]: - [ ] Add empty line between blocks

Comment (tictools) [inline index.js:35]: Avoid binding constant names to its implementation. (ul in this case, better list for example)

- [ ] Improve naming

File: index.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ul.appendChild(newItem4);
+
+// EXERCISE 6: Removing Elements
+// Remove the last list item from the unordered list
+
+let lastItem = document.getElementsByTagName("li")[3];
```
Comment (tictools) [inline index.js:41]:
```suggestion
let lastItem = $list.lastElementChild;
```

File: index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        divContent.style.display = "block"
+    } else {
+        divContent.style.display = "none";
+    }
+});
+
+// Exercise 10: Modifying Attributes
+// Select the input field and change its placeholder text to "Enter your name".
+
+inputText.placeholder = "Enter your name";
```
Comment (tictools) [inline index.js:77]: - [ ] Add empty line

**Review by tictools — APPROVED** [review]

**Pull request merged**